packages/block-editor/src/components/block-list/style.scss

```diff
@@ -464,6 +451,23 @@
 		.block-editor-block-mover.is-visible + .block-editor-block-list__breadcrumb {
 			top: (-$block-padding - $block-left-border-width - ($grid-size-small / 2));
 		}
+
+		// Align block toolbar to floated content.
+		@include break-mobile() {
+			.block-editor-block-toolbar {
+				/*!rtl:begin:ignore*/
+				left: $block-side-ui-width * 3;
```

Comment:
This needs just a tiny bit more space to avoid the arrows bumping up against the toolbar around 600-610px screen sizes. Specifically, we need exactly `6px` of extra space:

```suggestion
				left: $block-side-ui-width * 3 + ($grid-size-small * 1.5);
```

It's kind of annoying to use that math on `$grid-size-small` there, so alternatively we could just use `$grid-size` (8px) and have a couple extra pixels of spacing — it's better to have 2 extra pixels than 2 few pixels. 

**Before:** 

<img width="604" alt="Screen Shot 2019-07-15 at 7 46 05 AM" src="https://user-images.githubusercontent.com/1202812/61213977-a13e4100-a6d4-11e9-8a17-fdee27969073.png">


**After:** 

<img width="603" alt="Screen Shot 2019-07-15 at 7 41 54 AM" src="https://user-images.githubusercontent.com/1202812/61213982-a4393180-a6d4-11e9-8200-4a4fabae34b4.png">